src/main/cpp/t06_different.cpp

```diff
@@ -19,5 +19,28 @@
 using namespace std;
 
 int t06_different() {
-
+    int n,counter,k;
+    bool f;
+    cin >> n;
+    k = 1;
+    int a[n];
+    int b[k];
+    counter = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'counter' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493301)